### ISSUE TITLE
About visual_GPS

### ISSUE THREAD

**Issue by Ishimuraa** [AutoModelCar/AutoModelCarWiki#32]
Hi, I'd like to know if I can get the coordinates (and how) with visual_GPS directly or do I need to create an algorithm.

**Comment by AutoModelCar** [comment]
Yes, I don't know which version do you use, but you can find the topics list by 
"rostopic list" and if "/visual_gps/odom" is available you can subscribe to it. 
```sh
 rostopic echo /visual_gps/odom
```
You can look at the implementation below: 

https://github.com/AutoModelCar/catkin_ws_user/tree/master/src/fub_visual_gps
